From 704a97ecad2c081038001de140d0f11fc4857cdb Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Fri, 12 Oct 2012 16:47:28 +0800
Subject: [PATCH 2140/2508] ENGR00229350 Capture: ov5640 mipi: duplicated
 define of static variable

remove duplicated definition of prev_sysclk

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 drivers/media/video/mxc/capture/ov5640_mipi.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/media/video/mxc/capture/ov5640_mipi.c b/drivers/media/video/mxc/capture/ov5640_mipi.c
index cfb863e..d5f0773 100644
--- a/drivers/media/video/mxc/capture/ov5640_mipi.c
+++ b/drivers/media/video/mxc/capture/ov5640_mipi.c
@@ -705,7 +705,7 @@ static s32 ov5640_read_reg(u16 reg, u8 *val)
 	return u8RdVal;
 }
 
-static int prev_sysclk, prev_HTS, prev_sysclk;
+static int prev_sysclk, prev_HTS;
 static int AE_low, AE_high, AE_Target = 52;
 static int XVCLK = 2200;
 
-- 
1.7.7.4

